waku/v2/node/jsonrpc/filter_api.nim

```diff
@@ -16,10 +17,20 @@ proc installFilterApiHandlers*(node: WakuNode, rpcsrv: RpcServer) =
     messageCache = initTable[ContentTopic, seq[WakuMessage]]()
```

Comment:
Might want to pull out the type above. Also, here are some other LRU cache stuff that might be interesting https://github.com/status-im/nimbus-eth2/pull/1007/files and https://github.com/status-im/nim-libp2p/pull/360 in case you haven't seen them. I think the current implementation is fine for now, but just to be aware of

Reply:
Thanks, will do and will take a look. At first glance seems very relevant to what I want to achieve.

Is pulling out the type purely to increase readability/maintainability?

Reply:
Done in da7df23

Reply:
Mostly intuition in terms of where there might be some overlap in the future, i.e. highly likely we'll want to have a cache in other places. If it is a separate type it might be easier to spot/reason about/abstract later (e.g. using efficient and tested LRU impl later on). Not a must though.

Reply:
Makes sense. Thanks.